### ISSUE TITLE
Debug logging improperly

### ISSUE THREAD

**Issue by calebschoepp** [suborbital/reactr#79]
It appears that `log::debug` is incorrectly flagging itself as info. With the following code:

```rust
impl Runnable for HelloWorld {
    fn run(&self, input: Vec<u8>) -> Result<Vec<u8>, RunErr> {
        let in_string = String::from_utf8(input).unwrap();

        log::info("This is logged by INFO.");
        log::warn("This is logged by WARN.");
        log::error("This is logged by ERROR.");
        log::debug("This is logged by DEBUG.");

        Ok(String::from(format!("hello {}", in_string))
            .as_bytes()
            .to_vec())
    }
}
```

I get the following logging:

![Screenshot from 2021-05-05 20-09-48](https://user-images.githubusercontent.com/20216739/117232344-2fba1000-adde-11eb-9c58-e4a5696aff82.png)

Note the `(I)` in `"(I) This is logged by DEBUG."`

I think it is correct to file this in `reactr` because the WASM API code lives here but happy to move the issue.

**Comment by cohix** [comment]
You're right about where to file this....

At first glance the code handling this seems to be correct, so not quite sure what's going on: https://github.com/suborbital/reactr/blob/main/rwasm/api_log.go#L43

The debug level was added fairly recently so the version of the host code (linked here) may have been merged after the `0.2.0` release of Atmo. Perhaps try the newest Atmo version `0.2.2`?

**Comment by calebschoepp** [comment]
Okay, not sure why but this appears to be fixed on `0.2.2` like you suggested.

When I build an image with `atmo:v0.2.2` and set the appropriate log level `docker run -p 8080:8080 -e ATMO_HTTP_PORT=8080 -e ATMO_LOG_LEVEL=debug com.suborbital.test:v0.0.1 | jq`, I'm correctly seeing `(D)` for the debug logs.

Interestingly, `ATMO_LOG_LEVEL` has no affect on what `atmo:v0.2.0` decides to log.

Bottom line I guess this isn't an issue anymore because it is working in `atmo:v0.2.2`.

**Comment by cohix** [comment]
Yeah a number of logging changes were made right after 0.2.0, so that
checks out.

**Comment by ospencer** [comment]
@calebschoepp let us know if you see any more issues with this!

**Comment by calebschoepp** [comment]
Nope, think it's resolved
